layers/LAYERS16x.md

```diff
@@ -0,0 +1,23 @@
+Layers runtime: nodejs16.x
+----
+| Region | ARN |
+| --- | --- |
```

Comment:
Did you create it manually? You need to find where we creating this file

Reply:
correct, sorry i've removed it (I saw that node12 is identical to node 14 and assumed) 

lumigo-node/scripts/prepare_layer_files.sh

i think i found how to do it in the circleci builds.